pyramid_restful/api.py

```diff
@@ -194,10 +194,17 @@ def handle_error(self, request):
 
         request.response.status = '{0} {1}'.format(code, status)
 
-        return {
-            'type': projex.text.underscore(projex.text.underscore(type(err).__name__)),
-            'error': projex.text.nativestring(err)
-        }
+        # for 500 errors, only return server error
+        if code / 100 == 5:
+            return {
+                'type': 'server_error',
+                'error': 'An unknown server error occurred.'
+            }
```

Comment:
500 level errors should really be for the backend eyes only - all errors to communicate to the frontend should be 400 level.